templates/default/cut_tiles.sh.erb

```diff
@@ -13,6 +13,13 @@ trap 'rm $LOCK_FILE' EXIT 1 2 3
 export PATH=$PATH:/usr/local/bin
 export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib
 
+tile_dir=$(jq -r '.mjolnir.hierarchy.tile_dir' <%= node[:valhalla][:config] %>)
+
+#remove the hierarchy dirs.  if we crashed they are left around.
+rm -rf ${tile_dir}/0
```

Comment:
this will break everything. none of the machines that get tiles will ever get an update if you do this. below the process that tars them up and sends them to s3 relies on them still being there, thats why they arent cleared out each time. what you should instead do is set -e before pbfgraphbuilder, then check the return code of it and clear the dirs if the return code is non-zero. if you deploy it like this everything will stop working :smile: